geomstats/backend/numpy/__init__.py

```diff
@@ -158,12 +159,14 @@ def to_ndarray(x, to_ndim, axis=0):
     x = _np.array(x)
     if x.ndim == to_ndim - 1:
         x = _np.expand_dims(x, axis=axis)
-    assert x.ndim >= to_ndim
+
+    if x.ndim != 0:
+        assert x.ndim >= to_ndim
     return x
 
 
-def norm(val, axis):
-    return _np.linalg.norm(val, axis=axis)
+def norm(val, axis, keepdims=False):
```

Comment:
This should be in the file `/numpy/linalg.py` and called as `gs.linalg.norm` in the code.